tracer/test/snapshots/HotChocolate12Tests.SubmitsTraces.verified.txt

```diff
@@ -0,0 +1,345 @@
+﻿[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: aspnet_core.request,
+    Resource: GET /graphql/{*slug},
+    Service: Samples.HotChocolate,
+    Type: web,
+    Tags: {
+      aspnet_core.endpoint: Hot Chocolate GraphQL Pipeline,
+      aspnet_core.route: /graphql/{**slug},
+      component: aspnet_core,
+      env: integration_tests,
+      http.method: GET,
+      http.request.headers.host: localhost:00000,
+      http.status_code: 200,
+      http.url: http://localhost:00000/graphql,
+      language: dotnet,
+      runtime-id: Guid_1,
+      span.kind: server,
+      version: 1.0.0
+    },
+    Metrics: {
+      process_id: 0,
+      _dd.agent_psr: 1.0,
+      _dd.top_level: 1.0,
+      _dd.tracer_kr: 1.0,
+      _sampling_priority_v1: 1.0
+    }
+  },
+  {
+    TraceId: Id_1,
+    SpanId: Id_3,
+    Name: graphql.execute,
+    Resource: Query operation,
+    Service: Samples.HotChocolate-hotchocolate,
+    Type: graphql,
+    ParentId: Id_2,
+    Tags: {
+      component: HotChocolate,
+      env: integration_tests,
+      graphql.operation.type: Query,
+      graphql.source:
```

Comment:
I don't know if we're allowed line breaks in the tag, I have a feeling it will break things 🤔 The graphql.NET one for example looks like this:

```
graphql.source: query{hero{name appearsIn}},
```